locallib.php

```diff
@@ -459,7 +459,7 @@ public function get_views($query = '') {
                                       array('users' => array( array($field => $username,
                                                                     'query' => $query))));
             $result = array_pop($result);
-            $result['views']['ids'] = array_map('intval', explode(',', $result['views']['ids']));
+            $result['views']['ids'] = array_map('intval', explode(', ', $result['views']['ids']));
```

Comment:
This change causes errors on the submission page under certain circumstances, why is it here?
I suspect this is a static analyzer making a mistake?